Modules/Video/Core/test/itkTemporalProcessObjectTest.cxx

```diff
@@ -194,7 +194,7 @@ class CallRecord
  * Static list of CallRecord items representing the stack trace of
  * calls to GenerateData and TemporalStreamingGenerateData
  */
-std::vector<CallRecord> m_CallStack;
+static std::vector<CallRecord> m_CallStack;
```

Comment:
It's orthogonal to this change, but this `m_CallStack` appears to be a plain old global variable, not a class instance variable.  Should it have the `m_` prefix?

Reply:
It should not have the "m_" prefix.  Out of scope for this PR.